From 74d9de2dff7a6de7207fa383b306a9124c545628 Mon Sep 17 00:00:00 2001
From: Thomas Hipp <thomas.hipp@canonical.com>
Date: Tue, 3 Jan 2023 11:58:46 +0100
Subject: [PATCH] lxd-migrate: Fix usage string

This changes the usage string to clarify that lxd-migrate is an
interactive tool, and doesn't use positional arguments.

Resolves #11231.

Signed-off-by: Thomas Hipp <thomas.hipp@canonical.com>
---
 lxd-migrate/main_migrate.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lxd-migrate/main_migrate.go b/lxd-migrate/main_migrate.go
index 9f055cab15e7..058b6f9312bb 100644
--- a/lxd-migrate/main_migrate.go
+++ b/lxd-migrate/main_migrate.go
@@ -36,7 +36,7 @@ type cmdMigrate struct {
 
 func (c *cmdMigrate) Command() *cobra.Command {
 	cmd := &cobra.Command{}
-	cmd.Use = "lxd-migrate <target URL> <instance name> <filesystem root|raw image> [<filesystem mounts>...]"
+	cmd.Use = "lxd-migrate"
 	cmd.Short = "Physical to instance migration tool"
 	cmd.Long = `Description:
   Physical to instance migration tool
